packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/network/dedicatedCloud-datacenter-network.controller.js

```diff
@@ -1,6 +1,55 @@
-export default class {
+import { DATACENTER_NETWORK_SITE_WEB_LINK } from './dedicatedCloud-datacenter-network.constants';
+import { NETWORK_LABEL } from '../../../../dedicatedCloud/datacenter/dedicatedCloud-datacenter.constants.js';
+
+export default class DedicatedCloudDatacenterNetworkTab {
   /* @ngInject */
-  constructor($translate) {
+  constructor($translate, ovhManagerPccDatacenterService, coreConfig) {
+    this.coreConfig = coreConfig;
     this.$translate = $translate;
+    this.ovhManagerPccDatacenterService = ovhManagerPccDatacenterService;
+
+    this.NETWORK_LABEL = NETWORK_LABEL;
+  }
+
+  $onInit() {
+    this.fetchNsxEdgeReference();
+    this.loadComsumptionOfOption();
+    const user = this.coreConfig.getUser();
+
+    this.ovhSubsidiary = user.ovhSubsidiary;
+
+    this.guideUrl =
+      DATACENTER_NETWORK_SITE_WEB_LINK[this.ovhSubsidiary] ||
+      DATACENTER_NETWORK_SITE_WEB_LINK.DEFAULT;
+  }
```

Comment:
You don't need to "bind" the sub on the controller as you don't use it in your html template
```suggestion
    const {ovhSubsidiary} = this.coreConfig.getUser();

    this.guideUrl =
      DATACENTER_NETWORK_SITE_WEB_LINK[ovhSubsidiary] ||
      DATACENTER_NETWORK_SITE_WEB_LINK.DEFAULT;
  }
```